meshroom/ui/qml/Controls/TextFileViewer.qml

```diff
@@ -203,7 +203,7 @@ Item {
                     Rectangle {
                         width: 4
                         Layout.fillHeight: true
-                        color: duration > 0 ? Colors.durationColorScale(duration) : "transparent"
+                        color: Colors.interpolate(Colors.grey, Colors.red, duration/180)
```

Comment:
Need to use Colors.durationColorScale to get something visible. Just add interpolation in it.